### ISSUE TITLE
Reanimated 2 failed to create a worklet, maybe you forgot to add Reanimated's babel plugin?

### ISSUE THREAD

**Issue by Shasikhan** [gorhom/react-native-scroll-bottom-sheet#810]
# Bug

Following error is happening, I followed every step to install it and its dependencies.

Error: Requiring module "node_modules/@gorhom/bottom-sheet/src/index.ts", which threw an exception: Error: Reanimated 2 failed to create a worklet, maybe you forgot to add Reanimated's babel plugin?, js engine: hermes 

## Environment info

@gorhom/bottom-sheet            4.1.5 
react-native                                0.66.4 
react-native-reanimated           2.3.1  
react-native-gesture-handler   2.1.0  

<img width="379" alt="Screenshot 2022-01-11 at 4 04 01 PM" src="https://user-images.githubusercontent.com/32072222/148932514-7a620c5c-0fa1-4a45-be2a-d242c90cff9d.png">

**Comment by pszemang** [comment]
I had the same problem, [this](https://github.com/software-mansion/react-native-reanimated/issues/1875#issuecomment-815683730) solution helped me.

**Comment by festusasiyanbi** [comment]
I was faced with a similar issue but my bottom sheet now works without complaints after taking a few steps. I have highlighted the steps I took below

Step 1: Ensure react-native-reanimated, react-native-gesture-handler, and @gorhom/bottom-sheet are installed by running:

```
npm list react-native-reanimated react-native-gesture-handler @gorhom/bottom-sheet
```
- Run this command to install all using npm: 

```
npm install react-native-reanimated react-native-gesture-handler @gorhom/bottom-sheet
```
- Run this command to install all using yarn: 
```
yarn add react-native-reanimated react-native-gesture-handler @gorhom/bottom-sheet
```
- Run this command to install all in expo: 
```
npx expo install react-native-reanimated react-native-gesture-handler @gorhom/bottom-sheet.
```

Step 2: import react-native-reanimated and react-native-gesture-handler at the top of your index.js file(main entry point) as shown below
Command: 
```jsx
import 'react-native-reanimated';
import 'react-native-gesture-handler';
```
![Screenshot 2024-07-16 at 14 54 03](https://github.com/user-attachments/assets/5efbcd25-6958-4fa3-ada8-3ea42c603749)

Step 3: Include the react-native-reanimated plugins in your babel.config.js file
Command: 
```jsx
plugins: [
    'react-native-reanimated/plugin',
  ]
```
![Screenshot 2024-07-16 at 15 30 39](https://github.com/user-attachments/assets/45819ed2-4979-404e-91ac-aaba50c13e56)


Step 4: To save your time, create a bash script to perform the clean up of the iOS and android configurations. This command will clean up both platform configurations, and refresh and re-start the metro
Name: [scriptname].sh
Command: 
```bash
cd android && ./gradlew clean && cd .. && cd ios && pod install && cd .. && npm start --cache
```
See the attachment below:
![Screenshot 2024-07-16 at 15 04 11](https://github.com/user-attachments/assets/4b6cc9e2-5081-42b3-b9b3-8cb6c34e596b)



As long as you follow these steps I have highlighted above, your program should work.
Next, head to [gorhom's website](https://ui.gorhom.dev/components/bottom-sheet/) to start exploring the bottom sheet documentations.

Good luck!
